README.md

```diff
@@ -18,3 +15,36 @@
         <img src="http://img.shields.io/badge/swift-4.1-brightgreen.svg" alt="Swift 4.1">
     </a>
 </center>
+
+# Steam Scrum
+
+**Steam Scrum** is a project to provide a simple interface to do Scrum Groomings.
+
+The aim of the Scrum part is to list User Stories and compute amount of Story points an US reach.
+
+There is two interfaces
+  * One for each developers to select his amount of story point for an User Story
+  * The other one in the middle of the meeting table who indicates which developer still voting and at the end of the vote show the average.
```

Comment:
"L'autre au milieu de la table de réunion qui indique quel developeur est encore en train de voter et à la fin du vote montre la moyenne"

un peu bizarre la formulation : "The other interface is designed to be shown to all participants during the poker planning, indicated which developer hasn't voted yet. At the end of the vote, the interface shows the computed average of all votes."